doc/api/crypto.md

```diff
@@ -2875,6 +2875,18 @@ added: v15.6.0
 
 The date/time until which this certificate is valid.
 
+### `x509.validFromDate`
```

Comment:
Could you alphabetize the list by moving `x509.validFromDate` to follow `x509.validFrom`?